### ISSUE TITLE
keras.utils.to_categorical(y, num_classes) question

### ISSUE THREAD

**Issue by dnd88** [mtobeiyf/audio-classification#5]
Hello,
I have a question.

# nn.py (under 34 line)

Convert label to onehot
y_train = keras.utils.to_categorical(y_train-1, num_classes=num_classes)
y_test = keras.utils.to_categorical(y_test-1, num_classes=num_classes)

Why did you use (y_train - 1), not just (y_train) when you call keras.utils.to_categorical()

**Comment by imfing** [comment]
The original labels are like 1, 2, 3 ...
While in the training, we prefer 0, 1, 2 ...
That's why we need to subtract it by 1
